decidim-core/db/migrate/20180508111710_add_accepted_tos_version_field_to_users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddAcceptedTosVersionFieldToUsers < ActiveRecord::Migration[5.1]
+  def up
+    add_column :decidim_users, :accepted_tos_version, :datetime
+    Decidim::Organization.find_each do |organization|
```

Comment:
Use fake classes in migrations, please:

```ruby
class Organization < ApplicationRecord
  self.table_name = "decidim_organizations"
end
```

Etc.

Reply:
Ok, I'll use fake classes